app/src/main/java/org/mozilla/tv/firefox/utils/FormattedDomainWrapper.kt

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.utils
+
+import android.app.Application
+import java.net.URI
+
+/**
+ * TODO
+ */
+class FormattedDomainWrapper(private val application: Application) {
```

Comment:
Unrelated to your changes, what do you think about this being called ReadableDomain? Formatted sounds so unhelpful.

Reply:
Hm.  I looked at the implementation, and although (as far as I know) we only use this to make it readable, the methods themselves are a lot more flexible than that.  I think I'd prefer to leave it as is, even though it's not a super descriptive name.